ui/app/templates/components/identity/popup-alias.hbs

```diff
@@ -1,6 +1,6 @@
 <PopupMenu @name="alias-menu">
   <Confirm as |c|>
-    {{#let this.params.firstObject as |item|}}
```

Comment:
`firstObject` is deprecated

Reply:
FWIW I ended up turning this rule off in `.template-lintrc` since we still have array prototype extensions enable.